src/vault/api_util.clj

```diff
@@ -0,0 +1,129 @@
+(ns vault.api-util
```

Comment:
Is there a rationale for not putting this under the client namespace? 
I think organisation and dependency would be more clear if it lived at: `src/vault/client/api_util.clj`

This is especially true when its primarily a set of subroutines only used by `amperity.client.http`.

Reply:
You're right, I think that'd be cleaner. I'll move it!